### ISSUE TITLE
Feature Request:Org chart - multiple parents

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#511]
```
What would you like to see us add to this API?
To be able to set more than one parent to the org unit.


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?

Org Chart

*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************


function drawVisualization() {
  // Create and populate the data table.
  var data = new google.visualization.DataTable();
  data.addColumn('string', 'Name');
  data.addColumn('string', 'Manager');
  data.addColumn('string', 'ToolTip');
  data.addRows(5);
  data.setCell(0, 0, 'Mike');
  data.setCell(0, 2, 'The President');
  data.setCell(1, 0,
      'Jim', 'Jim<br/><font color="red"><i>Vice President<i></font>');
  data.setCell(1, 1, 'Mike');
  data.setCell(2, 0, 'Alice');
  data.setCell(2, 1, 'Mike');
  data.setCell(3, 0, 'Bob');
  data.setCell(3, 1, 'Jim,Alice');
  data.setCell(3, 2, 'Bob Sponge');
  data.setCell(4, 0, 'Carol');
  data.setCell(4, 1, 'Bob');

  // Create and draw the visualization.
  new google.visualization.OrgChart(document.getElementById('visualization')).
      draw(data, {allowHtml: true});
}

```

Original issue reported on code.google.com by `ludko2` on 2011-01-27 07:42:03

**Comment by orwant** [comment]
```
Issue 509 has been merged into this issue.
```

Original issue reported on code.google.com by `metula` on 2011-01-27 08:19:47

**Comment by orwant** [comment]
```
Hi. 
   I see that the same feature was requested by one more person. Is there any idea
on when we should be able to see that feature ? Like couple of weeks or couple of months
or more than that. Just an idea 

Regards
Meghashyam
```

Original issue reported on code.google.com by `meg.pgpex` on 2011-01-27 18:16:12

**Comment by orwant** [comment]
```
I really need to be able to display an org chart where child nodes can have multiple
parents.  If this request will go through soon, it would really save my life.
```

Original issue reported on code.google.com by `ajcrites` on 2011-04-28 15:47:15

**Comment by orwant** [comment]
```
This is a duplicate request of issue 162.
```

Original issue reported on code.google.com by `Hossein.FLorida` on 2011-07-24 00:53:45

**Comment by orwant** [comment]
```
Right.

```

Original issue reported on code.google.com by `jinji.viz` on 2011-07-24 07:20:19

**Comment by orwant** [comment]
```
Hello,
Has somebody found an alternative solution with org chart or another API to have mutiple
parents ?
Regards,
```

Original issue reported on code.google.com by `david.maechler` on 2011-08-08 10:22:03

**Comment by orwant** [comment]
```
has sombody found alternative solution for org chart to have multiple parents
```

Original issue reported on code.google.com by `rupesh.vaishnav@iniksha.com` on 2012-02-23 13:03:48

**Comment by orwant** [comment]
```
I'm also looking for a solution to enable multiple parents with orgchart... so sad to
find that nothing seems to have been made for this since 2011...

Would really be usefull...
```

Original issue reported on code.google.com by `laurent.lassure` on 2013-04-15 15:09:44

**Comment by orwant** [comment]
```
Would love to see this enhancement. Having more than one parent will help in lot of
my implementations.

```

Original issue reported on code.google.com by `b.archana.raju` on 2014-01-02 17:56:57

**Comment by orwant** [comment]
```
Yes I am waiting on this. Really need this for this application I am building.
*
```

Original issue reported on code.google.com by `dbdjbah` on 2014-04-22 07:34:49

**Comment by orwant** [comment]
```
Hello guys, is there a solution for the multiple parents problem?
```

Original issue reported on code.google.com by `roobertssuccess` on 2014-12-28 19:17:15

**Comment by mao75** [comment]
even for me it would be useful

**Comment by lisa-itmgr** [comment]
Why was this closed without implementation?

**Comment by dlaliberte** [comment]
We can't necessarily implement everything that is requested, though this idea of a multi-parent 'org' chart is good, and it would be useful in a variety of alternative scenarios, not strictly related to primarily hierarchical organization structures.  However, it opens up many additional design issues, and it may be better addressed as a feature of a more general network chart.  

I'll reopen it, in any event, though I don't know when we will have resources to do anything one way or another.

**Comment by greycubesgav** [comment]
I understand this may open up many additional design issues, however there's already a well renowned standard for this sort of thing provided through [GraphViz](https://www.graphviz.org/), who's format was actually already implemented by Google in the previous version of [Google's GraphAPI -> GraphViz Charts](https://developers.google.com/chart/image/docs/gallery/graphviz)
However this option was depreciated and then turned off earlier this year. 

Can we just have the GraphViz Charts option added back in to the Google Charts API?
Thanks

**Comment by qpkorr** [comment]
Great solution to re-enable the GraphViz interface!
